packages/teleport/src/cluster/features/featureSupport.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2020 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Icons from 'design/Icon';
+import Support from 'teleport/cluster/components/Support';
+import { withFeature, FeatureBase } from 'teleport/components/withFeature';
+import Ctx from 'teleport/teleportContext';
+import cfg from 'teleport/config';
+
+class FeatureSupport extends FeatureBase {
+  Component = withFeature(this)(Support);
```

Comment:
you can remove "withFeature" wrapper as this is a dummy feature. "withFeature" adds a wrapper that knows how to handle feature loading and error states which is not needed in this case.

Reply:
removed